scripts/pipeline/create_all_mlss.pl

```diff
@@ -594,43 +595,74 @@ sub make_named_species_set_from_XML_node {
 my $current_version = software_version();
 my %methods_not_worth_reporting = map {$_ => 1} qw(SYNTENY ENSEMBL_ORTHOLOGUES ENSEMBL_PARALOGUES ENSEMBL_HOMOEOLOGUES ENSEMBL_PROJECTIONS CACTUS_HAL_PW GERP_CONSTRAINED_ELEMENT GERP_CONSERVATION_SCORE);
 
+sub mlss2hash {
+    my $self = shift;
```

Comment:
Could a variable name such as `$mlss` be used here (and elsewhere in the `mlss2hash` function)?
```suggestion
    my $mlss = shift;
```